azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -72,7 +72,7 @@ def callback():
             connect_complete.set()
             logger.info("Successfully connected to Hub")
 
-        self._pipeline.connect(callback=callback)
+        self._transport.connect(callback=callback)
         connect_complete.wait()
 
```

Comment:
> InboxEmpty needs to be replaced with something more appropriate [](start = 43, length = 64)

I think we should hold off on this until a separate PR that adds across the board client exception story

Reply:
I am unable to resolve this comment.  I will try again. 

---
In reply to: [288331321](https://github.com/Azure/azure-iot-sdk-python-preview/pull/96#discussion_r288331321) [](ancestors = 288331321)

Reply:
<bzzzzt> unable to resolve.  github bug?  codeflow bug?

---
In reply to: [288777099](https://github.com/Azure/azure-iot-sdk-python-preview/pull/96#discussion_r288777099) [](ancestors = 288777099,288331321)

Reply:
It's because this comment only shows up on Revision 4 or less, starting with 5 it's "unavailable" because the line it was on got deleted.

---
In reply to: [288777644](https://github.com/Azure/azure-iot-sdk-python-preview/pull/96#discussion_r288777644) [](ancestors = 288777644,288777099,288331321)